impactsExporter.py

```diff
@@ -0,0 +1,163 @@
+
+    def create_csv(self, filePath, columns, rows):
+        with open(filePath, 'wb') as f:
+            w = csv.writer(f)
+            w.writerow(columns)
+            w.writerows(rows)
+            f.close()
+
+    def mark_export_as_done(self, item, filePath):
+        if isinstance(item, Export):
+            item.status = 'done'
+            item.file_path = filePath
+            item.updated_at = utils.get_current_time()
+            db.session.commit()
+            db.session.refresh(item)
+
+    def mark_export_as_error(self, item):
+        if isinstance(item, Export):
+            item.status = 'error'
+            item.updated_at = utils.get_current_time()
+            db.session.commit()
+            db.session.refresh(item)
+
+    def run(self):
+        try:
+            item = None
+            self.check_required_arguments()
+            item = self.get_oldest_waiting_export(self.client.id)
+            self.update_export_status(item, 'handling')
+            impacts = self.get_client_impacts_between_application_dates(item.client_id, item.start_date, item.end_date)
+            formated_impacts = self.format_impacts(impacts)
+            filePath = self.generate_file_path(item)
+            self.create_csv(filePath, formated_impacts['columns'], formated_impacts['rows'])
+            self.mark_export_as_done(item, filePath)
+            self.logger.info('Impacts export for %s is available at %s' % (item.client_id, filePath))
+        except UserWarning as w:
+            self.mark_export_as_error(item)
+            self.logger.info(w)
+            sys.exit(0)
+        except Exception as e:
+            self.mark_export_as_error(item)
+            self.logger.debug(e)
+            sys.exit(1)
+
+    def format_impacts(self, impacts):
+
+        navitia = Navitia(default_settings.NAVITIA_URL, self.coverage, self.token)
+
+        local_tz = pytz.timezone(self.time_zone)
+        columns = impacts.keys()
+
+        rows = []
+        for sub_dict in impacts.fetchall():
+            row = []
+            for column in columns:
+                val = sub_dict[column]
+                if column == 'pt_object_name' :
+                    row.append(navitia.find_tc_object_name(sub_dict['pt_object_uri'], sub_dict['pt_object_type']))
+                elif isinstance(val, datetime.date):
+                    row.append(utils.utc_to_local(val, local_tz))
+                else:
+                    row.append(val)
+
+            rows.append(row)
+
+        return {'columns' : columns, 'rows' : rows}
+
+def get_command_arguments(argv):
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--client_id', help='Client UUID')
+    parser.add_argument('--coverage', help='Navitia coverage')
+    parser.add_argument('--token', help='Navitia token')
+    parser.add_argument('--tz', help='Time zone')
+
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    args = get_command_arguments(sys.argv[1:])
+
+    tz = args.tz;
```

Comment:
Unnecessary semicolon
```suggestion
    tz = args.tz
```